contrib/depends/packages/openssl.mk

```diff
@@ -51,7 +53,8 @@ endef
 
 define $(package)_preprocess_cmds
   sed -i.old "/define DATE/d" util/mkbuildinf.pl && \
-  sed -i.old "s|engines apps test|engines|" Makefile.org
+  sed -i.old "s|engines apps test|engines|" Makefile.org && \
+  sed -i -e "s/-mandroid //" Configure
```

Comment:
The `-mandroid` flag was needed for the old gcc-based Android toolchain. It's not recognized by clang.